src/Markup/Avalonia.Markup.Xaml/MarkupExtensions/OnPlatformExtension.cs

```diff
@@ -0,0 +1,129 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Avalonia.Metadata;
+using Avalonia.Platform;
+
+namespace Avalonia.Markup.Xaml.MarkupExtensions;
+
+public class On
+{
+    public string Platform { get; set; } = "Unknown";
+
+    [Content]
+    public object? Content { get; set; }
```

Comment:
Why is it `object`-typed while `OnPlatform<T>` is strongly typed?

Reply:
Fixed